test/e2e/kubeproxy.go

```diff
@@ -38,7 +40,6 @@ import (
 const (
 	endpointHttpPort        = 8080
 	endpointUdpPort         = 8081
-	endpointHostPort        = 8082
```

Comment:
Would you mind to set the NodeName correctly for each pod instead of this "port hack". Constant host ports are not nice.